### PR TITLE
[CUR-791] Child subject & tier subtitle on title page for docx

### PR DESCRIPTION
**Pull request by sonalisin**
## Description

Music year: {owa_music_year}

- Updated title page to have correct subtitle according to filters 
- Updated filename to have proper formatting (e.g. "Combined-science" becomes "Combined science") for subject titles longer than a word

## How to test

1. Go to {owa_deployment_url}
2. Click on \_\_\_\_\_\_
3. You should see \_\_\_\_\_\_

## Screenshots

How it used to look (delete if n/a):
Screenshot 2024-08-05 at 15 22 54
Screenshot 2024-08-05 at 15 25 09

How it should now look:
Screenshot 2024-08-05 at 15 22 05
Screenshot 2024-08-05 at 15 24 23
English 
Screenshot 2024-08-05 at 15 53 35

## Checklist

- [ ] Added or updated tests where appropriate
- [ ] Manually tested across browsers / devices
- [ ] Considered impact on accessibility
- [ ] Design sign-off
- [ ] Approved by product owner
- [ ] Does this PR update a package with a breaking change

### CODE REVIEW

**Comment by rightsaidjames** [comment]
@sonalisin Can I have a screenshot of a Secondary document for another subject that doesn't have child subjects or tiers (e.g. English?) Other than that, all looks good!

**Review by rightsaidjames — APPROVED** [review]

File: src/pages-helpers/curriculum/docx/builder/1_frontCover.ts [hunk 1]
```diff
@@ -20,34 +22,44 @@ import { getSubjectIconAsset } from "@/image-data";
...
+  { data, slugs }: { data: CombinedCurriculumData; slugs: Slugs },
 ) {
   const iconKey = data.subjectTitle.toLowerCase();
 
   const sanityUrl = getSubjectIconAsset(iconKey)?.url;
   const images = await insertImages(zip, {
     icon: sanityUrl
       ? makeTransparentIfSanity(sanityUrl, cmToPxDpi(13))
-      : join(
+      : relative(
```
Comment (orangemug) [inline src/pages-helpers/curriculum/docx/builder/1_frontCover.ts:33]: Any reason for this change? I think this'll break in prod

Comment (sonalisin) [reply src/pages-helpers/curriculum/docx/builder/1_frontCover.ts:33]: No, think it was a conflict from earlier! Will change it back

File: src/pages-helpers/curriculum/docx/builder/1_frontCover.ts [hunk 2]
```diff
@@ -44,10 +46,20 @@ export default async function generate(
 
   const phaseTitle = keyStageFromPhaseTitle(data.phaseTitle);
   const subjectTitle = data.subjectTitle;
-  const examboardTitle = data.examboardTitle
-    ? `${data.examboardTitle} (KS4)`
+
+  const examboardTitle = data.examboardTitle ? `${data.examboardTitle}` : "";
+  const tierTitle = slugs.tierSlug ? `${capitalize(slugs.tierSlug)}` : "";
```
Comment (orangemug) [inline src/pages-helpers/curriculum/docx/builder/1_frontCover.ts:51]: Is there not a `*Title` we can use for these?

Comment (orangemug) [reply src/pages-helpers/curriculum/docx/builder/1_frontCover.ts:51]: I think `mv_lesson_listing_5_0_1` gets `tierTitle` as a reference.

Comment (sonalisin) [reply src/pages-helpers/curriculum/docx/builder/1_frontCover.ts:51]: Creating a tech debt ticket for this here for future child subjects: https://www.notion.so/oaknationalacademy/DOCX-Child-subject-tier-filtering-title-on-title-page-should-come-from-MV-data-5f340bc9b3024de38be4adc5b770c938?pvs=4

**Pull request merged**